src/Xrefcheck/Config.hs

```diff
@@ -50,7 +50,7 @@ data VerifyConfig = VerifyConfig
 -- would be lost.
 defConfigText :: ByteString
 defConfigText =
-  $(TH.lift =<< qReadFileBS ("src-files" </> "def-config.yaml"))
+  $(embedFile ("src-files" </> "def-config.yaml"))
```

Comment:
So it happened that `qReadFileBS` works bad on Windows, but `embedFile` does fine?

Reply:
Yes. The reason is that `qReadFileBS` gets an absolute path from `iserv`, which for windows cross-compilation runs inside wine, thus producing a path like `Z:\build\xrefcheck\src-files`, which makes no sense on Linux. `embedFile`, on the other hand, works with relative paths AFAIU and doesn't talk to `iserv` at all, so it works fine.

Reply:
Cool, nice that you managed to find a workaround for this.

Reply:
Does it mean that we should always prefer `embedFile` to `qReadFileBS`?

Reply:
@gromakovsky yes, if we want to support cross-compilation to windows, we shouldn't use `qReadFileBS`, because it simply won't work.

Reply:
I originally used `qReadFileBS` because it states its return type explicitly (`Q ByteString` instead of `Q Exp`), but that seems not that important taking cross-platform issues into account)